tests/integration/test_manifest_generation.py

```diff
@@ -0,0 +1,1217 @@
+
+        assert (
+            sheet1[f"{columns['Filename']}4"].value
+            == "Manifest generation - Manual test - generate an existing manifest/test dataset/test dataset 3/sample C.txt"
+        )
+        assert sheet1[f"{columns['Sample ID']}4"].value is None
+        assert sheet1[f"{columns['File Format']}4"].value is None
+        assert sheet1[f"{columns['Component']}4"].value == "BulkRNA-seqAssay"
+        assert sheet1[f"{columns['Genome Build']}4"].value is None
+        assert sheet1[f"{columns['Genome FASTA']}4"].value is None
+        assert sheet1[f"{columns['entityId']}4"].value == "syn63561085"
+
+        # AND there are no more columns in the sheet
+        assert sheet1[f"{columns['entityId']}1"].offset(column=1).value is None
+
+        # AND the first row is locked on scroll
+        assert sheet1.freeze_panes == "A2"
+
+        # AND each of these cells in the first row has a comment "TBD"
+        for col in [
+            "Filename",
+            "Sample ID",
+            "File Format",
+            "Component",
+            "Genome Build",
+            "Genome FASTA",
+        ]:
+            assert sheet1[f"{columns[col]}1"].comment.text == "TBD"
+
+        # AND the entityId column in the first row does not have a comment
+        assert sheet1[f"{columns['entityId']}1"].comment is None
+
+        # AND the dropdown lists exist and are as expected
+        data_validations = sheet1.data_validations.dataValidation
+        file_format_validation = None
+        genome_build_validation = None
+        for dv in data_validations:
+            if f"{columns['File Format']}2" in dv.sqref:
+                file_format_validation = dv
+                continue
+            elif f"{columns['Genome Build']}2" in dv.sqref:
+                genome_build_validation = dv
+                continue
+            # AND there are no other data validations
+            assert False, f"Unexpected data validation found: {dv}"
+
+        assert file_format_validation is not None
+        assert file_format_validation.type == "list"
+        assert (
+            file_format_validation.formula1
+            == f"Sheet2!${columns['File Format']}$2:${columns['File Format']}$5"
+        )
+
+        assert genome_build_validation is not None
+        assert genome_build_validation.type == "list"
+        assert (
+            genome_build_validation.formula1
+            == f"Sheet2!${columns['Genome Build']}$2:${columns['Genome Build']}$5"
+        )
+
+        # AND the fill colors are as expected
+        for col in ["Filename", "Sample ID", "File Format", "Component"]:
+            assert sheet1[f"{columns[col]}1"].fill.start_color.index == LIGHT_BLUE
+
+        for col in [
+            "Genome Build",
+            "Genome FASTA",
+            "entityId",
+        ]:
+            assert sheet1[f"{columns[col]}1"].fill.start_color.index == GRAY
+
+        # AND conditional formatting is functioning as expected (MANUAL VERIFICATION)
```

Comment:
This comment is a little confusing to me. Is it referencing the tests right below it?

Reply:
It is setting a value right below this line:
`workbook["Sheet1"][f"{columns['Diagnosis']}2"].value = "Cancer"`

that must be manually verified for conditional formatting. The library does not re-render the fill color of cells so it's a part of the instructions in the docstring of things to verify. Since we wanted to make manual verification as simple as "Open this and look at this", we're setting the values ahead of time so no one needs to mess around with changing values in the cells themselves. The test does it for them.